pyCGM_Single/pyCGM.py

```diff
@@ -536,7 +536,7 @@ def ankleJointCenter(frame,knee_JC,delta,vsk=None):
     -------
     >>> import numpy as np 
     >>> import math
-    >>> from .pyCGM import *
+    >>> from .pyCGM import ankleJointCenter, findJointC, cross, norm2d
```

Comment:
Why are you importing cross and norm2d?

Reply:
I imported all the relevant other functions used in each individual function just to be safe.

Reply:
For the doctests, the safer thing I think for us is to only import the function being used, not the functions that the one we are using also uses.

Reply:
Ah I understand, I resolved all the functions to only import numpy (when needed) and the functions called in the doctests themselves.